lib/core/src/Cardano/Wallet/DB/Sqlite.hs

```diff
@@ -763,8 +767,9 @@ newDBLayer trace defaultFieldValues mDatabaseFile ti = do
                 insert_ (mkWalletEntity wid meta)
             when (isRight res) $ do
                 insertCheckpoint wid cp <* writeCache wid (Just cp)
-                let (metas, txins, txouts, ws) = mkTxHistory wid txs
-                putTxs metas txins txouts ws
+                let (metas, txins, txouts, txoutTokens, ws) =
```

Comment:
> What about renaming txoutTokens → txoutToks rather than breaking the line?

As we already use `token` and `tokens` in other places, my preference would be to keep this name consistent with the usage in other places.

We _could_ rename `txoutTokens` to just `tokens`, but I think it's also nice for the name to indicate that these tokens are related to `txouts`.

Reply:
OK.